adhocracy4/comments_async/serializers.py

```diff
@@ -14,16 +14,24 @@ class CommentSerializer(serializers.ModelSerializer):
     user_profile_url = serializers.SerializerMethodField()
     user_image = serializers.SerializerMethodField()
     is_deleted = serializers.SerializerMethodField()
-    ratings = serializers.SerializerMethodField()
+    ratings = serializers.ModelSerializer(many=True)
     is_moderator = serializers.SerializerMethodField()
+    comment_categories = serializers.ModelSerializer(many=True)
+    child_comments = serializers.ModelSerializer(many=True)
 
     class Meta:
         model = Comment
         read_only_fields = ('modified', 'created', 'id',
                             'user_name', 'user_pk', 'user_image',
                             'user_image_fallback', 'ratings',
                             'content_type', 'object_pk')
-        exclude = ('creator',)
+        fields = ('id', 'user_name', 'user_pk', 'user_profile_url',
+                  'user_image', 'is_deleted', 'ratings',
+                  'is_moderator', 'child_comments',
+                  'created', 'modified', 'object_pk', 'comment',
+                  'is_removed', 'is_censored', 'is_blocked',
+                  'comment_categories', 'last_discussed',
+                  'is_moderator_marked', 'content_type')
```

Comment:
oh and then remove the child_comments here if you add them in Meta of ThreadSerializers